README.md

```diff
@@ -122,6 +122,14 @@ A query to select browsers (ex: last 2 versions, > 5%) using [browserslist](http
 
 Note, browsers' results are overridden by explicit items from `targets`.
 
+#### Browserslist config file.
```

Comment:
I think it's useful for this section to describe how `browserslist` config interops with `targets`.

For example how it merges (or doesn't, depending on what we decide).

Then, maybe we should move the `browserslist` default behavior into a main section above options since it's a really powerful default.

Thoughts?